src/framework/preferences/messages/textFont_pt_BR.json

```diff
@@ -1,5 +1,6 @@
 {
-    "textFont-default": "Padrão",
+    "textFont-default": "No Preference",
+    "textFont-system": "Device Font",
```

Comment:
@gtirloni would you be able to help translate these two strings?

Reply:
The recommendation from @gtirloni is:

No Preference => Sem Preferência
Device Font => Fonte do Dispositivo